1.8/minio/README.md

```diff
@@ -0,0 +1,97 @@
+# Install and use Minio on DC/OS
+
+[Minio](https://minio.io) is an object storage server released under Apache License v2.0. It is compatible with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
+
+Minio server is light enough to be bundled with the application stack, similar to NodeJS, Redis and MySQL. 
+
+This quickstart installation uses the single node version of Minio (Minio FS), checkout the [docs](https://docs.minio.io) for more details.
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down. 
+
+- Estimated time for completion: up to 5 minutes. 
+- Target audience:
+ - Operators
+ - Application admins
+ - Developers 
+ - Devops Engineers
+- Scope: Learn to install Minio on DCOS and learn to use it using minio command line tool `mc`.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Setting up Minio](#setting-up-minio)
+- [Using Browser Interface](#using-browser-interface)
+- [Install mc](#install-mc)
+- [Configure mc](#configure-mc)
+- [Resources](#resources)
+
+## Prerequisites
+
+- DC/OS 1.8 or later
+- [Marathon-lb](https://dcos.io/docs/1.8/usage/service-discovery/marathon-lb/usage/) must be installed and running
+- IP address of the public agent(s) where Marathon-lb or an available hostname configured to point to the public agent(s) where Marathon-lb is running.
+
+## Setting up Minio 
+
+1. Before starting, identify the IP address or hostname of a public agent where Marathon load balancer is running. 
+
+
+- Visit the Universe page in DC/OS, and click on the "Install Package" button underneath Minio.
+
+! [Install Minio](img/install.png)
+
+- Click on "Advanced Installation" and navigate to the "networking" tab. Specify the IP address or hostname of the public agent where marathon load balancer is running. Make sure you remove the leading http:// and the trailing / from the IP. 
```

Comment:
Please make sure that you're using Marathon-LB consistently (or shorten to MLB).